src/libstd/panicking.rs

```diff
@@ -263,6 +263,11 @@ pub mod panic_count {
             // Fast path: if `GLOBAL_PANIC_COUNT` is zero, all threads
             // (including the current one) will have `LOCAL_PANIC_COUNT`
             // equal to zero, so TLS access can be avoided.
+            //
+            // A relaxed atomic load is equivalent to a normal aligned memory read
+            // (e.g., a `mov` instruction in x86), while a TLS access might require
```

Comment:
It is not equivalent. The compiler can optimize relaxed less than it can optimize normal reads.

Reply:
What about "In terms of performance, a relaxed atomic load is similar to a normal aligned memory read (e.g., a `mov` instruction in x86), but with some compiler optimization restrictions."?

Reply:
That sounds great. :)